extra/build.sbt

```diff
@@ -11,6 +11,7 @@ libraryDependencies ++= {
     "ai.lum"     %% "nxmlreader"            % "0.1.2",
     "org.clulab" %% "processors-main"       % procVersion,
     "org.clulab" %% "processors-corenlp"    % procVersion,
+    "org.clulab" %% "reach-processors"      % "1.6.3-SNAPSHOT"
```

Comment:
I don't think we want to include a snapshot dependency in the default branch.  


@kwalcock , is a version going to be published?  What is the ETA on getting the transformer-based processor integrated into Reach?